src/main/java/com/flightstats/hub/webhook/WebhookError.java

```diff
@@ -37,80 +38,40 @@
 @Singleton
 class WebhookError {
     private final static Logger logger = LoggerFactory.getLogger(WebhookError.class);
-    private static final int MAX_SIZE = 10;
+    private static final String BASE_PATH = "/GroupError";
 
-    private final CuratorFramework curator;
+    private final SafeZooKeeperUtils zooKeeperUtils;
     private final ChannelService channelService;
+    private final ErrorNodeNameGenerator errorNameGenerator;
+    private final WebhookErrorReaper webhookErrorReaper;
 
     @Inject
-    public WebhookError(CuratorFramework curator, ChannelService channelService) {
-        this.curator = curator;
-        this.channelService = channelService;
+    public WebhookError(SafeZooKeeperUtils zooKeeperUtils, ChannelService channelService) {
```

Comment:
I think I do that in a later commit? Well, `WebhookErrorReaper`, anyway. The name generator stuff felt weird because I had really only extracted it into its own class so I could mock it for testing. Pretty sure normal people would have left it as a private method or something. I like the idea of having a single constructor, so I'll make it so.